### PR TITLE
Aad 229 base pwa apim setup

### PR DESCRIPTION
**Pull request by alireid**
Base APIM setup for apprentice-app

### CODE REVIEW

File: src/ApprenticeApp/SFA.DAS.ApprenticeApp.Api/ErrorHandler/ErrorHandlerExtensions.cs
Comment (CathyGroom) [inline src/ApprenticeApp/SFA.DAS.ApprenticeApp.Api/ErrorHandler/ErrorHandlerExtensions.cs:1]: Line 67 Whitespace

File: src/ApprenticeApp/SFA.DAS.ApprenticeApp/Application/Commands/ApprenticeAccounts/AddOrUpdateMyApprenticeshipCommandHandler.cs
Comment (CathyGroom) [inline src/ApprenticeApp/SFA.DAS.ApprenticeApp/Application/Commands/ApprenticeAccounts/AddOrUpdateMyApprenticeshipCommandHandler.cs:1]: Lines 63, 64 Whitespace

File: src/ApprenticeApp/SFA.DAS.ApprenticeApp/Services/CoursesService.cs
Comment (CathyGroom) [inline src/ApprenticeApp/SFA.DAS.ApprenticeApp/Services/CoursesService.cs:1]: Line 49 Whitespace

Comment (CathyGroom) [inline src/ApprenticeApp/SFA.DAS.ApprenticeApp/Services/CoursesService.cs:1]: Line 49 Whitespace

**Review by HussainsGitHub — APPROVED** [review]

**Review by CathyGroom — APPROVED** [review]
Looks good to me.

**Review by divesh987 — APPROVED** [review]

**Pull request merged**